search_bar.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style1.css">
+
+</head>
+<body>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+        <label>
+            <div class="input">
```

Comment:
```suggestion
    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
      <label>
        <div class="input">
```
Дуже багато відступів. Один рівень вкладеності - це два пробіла. Давай це скрізь пофіксимо
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation